sqlboiler/sqlboiler.go

```diff
@@ -0,0 +1,57 @@
+package sqlboiler
+
+import (
+	"path"
+	"strings"
+
+	"github.com/aserto-dev/clui"
+	"github.com/aserto-dev/mage-loot/deps"
+	"github.com/magefile/mage/sh"
+)
+
+type sqlboilerArgs struct {
+	args []string
+}
+
+// Arg represents a sqlboiler CLI argument
+type Arg func(*sqlboilerArgs)
+
+var (
+	ui = clui.NewUI()
+)
+
+// Run runs the sqlboiler CLI
+func Run(args ...Arg) error {
+	sqlboilerArgs := &sqlboilerArgs{}
+
+	for _, arg := range args {
+		arg(sqlboilerArgs)
+	}
+
+	finalArgs := []string{}
+
+	finalArgs = append(finalArgs, sqlboilerArgs.args...)
+
+	psqlPluginPath := path.Dir(deps.GoBinPath("sqlboiler-psql"))
```

Comment:
Do we need to bind this to `sqlboiler-psql`? Can it be the generic `sqlboiler`?

Reply:
We need to tell sqlboiler where the psql plugin is.